apps/emqx_bridge/src/emqx_bridge_v2.erl

```diff
@@ -1575,7 +1871,7 @@ actions_convert_from_connectors(Conf) ->
         Conf
     ).
 
```

Comment:
After spending some time looking into `emqx_action_info:action_convert_from_connector` I'm still not sure what it is doing. However, I think we can improve this naming or add a comment in later PR since improving the readability of this is unrelated to the main purpose of this PR.